### DISCUSSION TITLE
No curly braces from macOS IT keyboard

### DISCUSSION THREAD

**Discussion by s7maestro** [nesbox/TIC-80#2243] [Q&A]
I'm literally prevented from using curly brackets in the code editor on a MacOS with an IT keyboard layout (QWERTY with accented vowels on the right).

To write curly braces with this MacOS keyboard, I have to Alt-Shift-è or Alt-Shift-+ (EDIT: è and + on an IT keyboard are located respectively on the [ and ] buttons of a US keyboard, on the left of the Enter button) usually, but using this combination inside TIC80 does nothing and this way I'm literally prevented from writing tables

**Comment by Skeptim** [comment]
This has been solved with the latest release.
